tests/data/graphs/fit_curve.json

```diff
@@ -113,9 +113,10 @@
              "arrayelement3": {
                "process_id": "array_element",
                "arguments": {
-                 "data": {
-                   "from_parameter": "parameters"
-                 },
+                 "data": [
+                  3,
+                  2
+                ],
                  "index": 2
```

Comment:
isn't index: 2 out of range for data = [3,2]? This probably doesn't matter to test the functionality :)

Reply:
Yea I forgot I changed that and never changed it back, it would matter if the compiled function from fit_curve would ever be executed, which it's not. We don't check for potential runtime errors when compiling, that's out of scope.

The actual testing of the functions execution happens in the new test!